dockerfiles/platform-gnn-gpu/Dockerfile

```diff
@@ -66,37 +74,43 @@ RUN pip3 install       \
 # TensorFlow with GPU support
 # Reference: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/dockerfiles/dockerfiles/gpu.Dockerfile
 
-ARG CUDA=10.1
-ARG CUDNN=7.6.4.38-1
-ARG CUDNN_MAJOR_VERSION=7
+ARG ARCH
+ARG CUDA=11.3
+ARG CUDNN=8.2.1.32-1
+ARG CUDNN_MAJOR_VERSION=8
 ARG LIB_DIR_PREFIX=x86_64
-ARG LIBNVINFER=6.0.1-1
-ARG LIBNVINFER_MAJOR_VERSION=6
+ARG LIBNVINFER=8.0.0-1
+ARG LIBNVINFER_MAJOR_VERSION=8
 
 # Needed for string substitution
 SHELL ["/bin/bash", "-c"]
 # Pick up some TF dependencies
-RUN apt-get update -y && apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
losing the -y here, any chance we're going to run into a prompt stopping the command from executing?

Reply:
fixed